packages/marshaller/src/dashy.ts

```diff
@@ -107,9 +107,15 @@ export class Dashy extends SplitPanel {
                 }
                 return this._elementContainer.refresh();
             }).then(() => {
+                let hasError = false;
                 for (const error of this._elementContainer.validate()) {
+                    hasError = true;
                     logger.warning(error.elementID + " (" + error.source + "):  " + error.msg);
                 }
+                if (hasError) {
+                    //  Re-render to highlight errrors  ---
+                    this._lhsDashboard.renderPromise();
```

Comment:
Does this somehow trigger a re-render for the right hand side as well?

Reply:
No (it will have been triggered by the setting of "active" on line 106